site/pages/faq.md

```diff
@@ -82,3 +82,7 @@ No, competencies are not weighted. No competency is more important than any othe
 ## Who is in charge of maintaining the competencies?
 
 A group of engineers from across Product &amp; Technology. See [The Engineering Progression Working Group docs](/docs/working-group/), or check the [GitHub team](https://github.com/orgs/Financial-Times/teams/engineering-progression-working-group/members) for individual members.
+
+## Guidelines for writing feedback or writing about competencies you have achieved
```

Comment:
I think we should either remove this from the FAQ (just leaving it in the "how to use" guide), or rephrase this to be a question. So far it's not a question that we've been asked a lot, so maybe not including it in the FAQ is the way to go. Adding the rest of the group as a reviewer so they get notified.

Reply:
OK we agreed that rephrasing this as a question and leaving it in the FAQs